apps/web/components/eventtype/EventLimitsTab.tsx

```diff
@@ -314,6 +347,7 @@ export const EventLimitsTab = ({ eventType }: Pick<EventTypeSetupProps, "eventTy
               childrenClassName="lg:ml-0"
               title={t("limit_total_booking_duration")}
               description={t("limit_total_booking_duration_description")}
+              {...durationLimitsLocked}
```

Comment:
This was removed recently to allow these fields to be unlocked for v1, now with v2 we need them as the team admin can discretely select their state